.importjs.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "environments": ["node"],
-    "declaration_keyword": "const",
+    "declaration_keyword": "import",
```

Comment:
I considered that, but I didn't because we have talked about defaulting to const for node env. #235